index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital@1&family=Merriweather:ital,wght@0,700;1,700&family=Oswald:wght@700&display=swap"
+      rel="stylesheet" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css"
+      integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css" />
+    <title>CV</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="content content--left">
+        <header class="content__header">
+          <img src="IMG_5236.jpg" alt="Profile Photo of Gevindu Hapuarachchi" />
+
+          <h1>Gevindu Hapuarachchi</h1>
+          <h2>Intern Software Engineer</h2>
+        </header>
+        <section class="content__social">
+          <h2>Contact Information</h2>
+          <hr />
+          <a
+            href="tel:01122321231"
+            class="link"
+            aria-label="Call Gevindu Hapuarachchi at 01122321231">
+            <i class="fa-solid fa-square-phone"></i><br />01122321231
+          </a>
```

Comment:
This link could look like this:

`
<a
            href="tel:01122321231"
            class="link"
            aria-label="Call Gevindu Hapuarachchi at 01122321231">
            <span class="fa-solid fa-square-phone" aria-hidden="true" focusable="false"></span>
            <br />
            <span class="visually-hidden">01122321231</span>
          </a>
`

where "visually-hidden" class is an extra css class. Have a look here: 
https://stackoverflow.com/questions/62703524/how-to-make-an-html-link-displayed-as-an-icon-accessible

Reply:
Done.